react/tic-tac-toe/src/redux/Matches/actions.js

```diff
@@ -0,0 +1,12 @@
+export const actions = {
+  GET_MATCHES: '@@MATCHES/GET_MATCHES'
+};
+
+const actionsCreators = {
+  getMatches: data => ({
```

Comment:
Redux-thunk lets you make async calls. For example
```js
const actionsCreators = {
  getMatches: async dispatch => ({
     dispatch( ... );
     const { data } = await MatchesService.getMatches();
     ...
   })
};
```
Remember to make the service calls inside this scope. Not in the component.